src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java

```diff
@@ -191,7 +197,21 @@ private static NettyChannelBuilder newNettyChannelBuilder(String targetUrl, Stri
    * @throws IOException in case the call credentials can't be constructed.
    */
   @Nullable
```

Comment:
Copy the comment from `newCallCredentials` here, to make it clear where we attempt to retrieve credentials from.